EventFilter/RPCRawToDigi/interface/RPCLBRecord.h

```diff
@@ -0,0 +1,71 @@
+#ifndef EventFilter_RPCRawToDigi_RPCLBRecord_h
+#define EventFilter_RPCRawToDigi_RPCLBRecord_h
+
+#include <stdint.h>
+
+class RPCLBRecord
+{
+public:
+    /* https://twiki.cern.ch/twiki/bin/viewauth/CMS/DtUpgradeTwinMux#RPC_payload RPC_optical_links_data_format.pdf */
+    /* 4b partition is split in 3b connector + 1b partition */
+    static ::uint32_t const bcn_mask_            = 0xfe000000;
```

Comment:
(minor) public const members more frequently do not have a trailing underscore in CMSSW parts that I recall